crates/autopilot/src/run_loop.rs

```diff
@@ -127,10 +131,18 @@ impl RunLoop {
         Some(domain::AuctionWithId { id, auction })
     }
 
-    async fn single_run(&self, auction_id: domain::auction::Id, auction: &domain::Auction) {
+    async fn single_run(
+        &self,
+        auction_id: domain::auction::Id,
+        auction: &domain::Auction,
+        init_block_timestamp: u64,
```

Comment:
nit: If this variable is only passed in to trigger the metric, we could probably also simply do it where we call `single_run`

Reply:
To fetch the latest block timestamp or what exactly? That would work as long as the auction update function takes less than 1 round. For arbitrum, this is not the case.